cmake/GzSetCompilerFlags.cmake

```diff
@@ -347,14 +348,24 @@ macro(_gz_setup_msvc)
   # framework.
   #
   # In some cases, a user might want to compile with the static runtime. This
-  # should ONLY be done if they do not intend to use the ignition library as
+  # should ONLY be done if they do not intend to use the Gazebo library as
   # part of a plugin-based framework.
-  option(IGN_USE_STATIC_RUNTIME "Use the static runtime (strongly discouraged)" OFF)
+
+  if(IGN_USE_STATIC_RUNTIME)  # TODO(CH3): Deprecated. Remove on tock.
+    message(WARNING "Deprecated. Use [GZ_USE_STATIC_RUNTIME] instead! Use the static runtime (strongly discouraged)")
+    set(GZ_USE_STATIC_RUNTIME ${IGN_USE_STATIC_RUNTIME})
+  else()
+    option(GZ_USE_STATIC_RUNTIME "Use the static runtime (strongly discouraged)" OFF)
+    set(IGN_USE_STATIC_RUNTIME ${GZ_USE_STATIC_RUNTIME})
+  endif()
+
   if(BUILD_SHARED_LIBS)
     # Users should not choose the static runtime unless they are compiling a
     # static library, so we completely disable this option if BUILD_SHARED_LIBS
     # is turned on.
-    set(IGN_USE_STATIC_RUNTIME OFF CACHE BOOL "Use the static runtime (strongly discouraged)" FORCE)
+    set(GZ_USE_STATIC_RUNTIME OFF CACHE BOOL "Use the static runtime (strongly discouraged)" FORCE)
+    set(IGN_USE_STATIC_RUNTIME ${GZ_USE_STATIC_RUNTIME}  # TODO(CH3): Deprecated. Remove on tock.
```

Comment:
If I understand correctly, this is internally setting the variable. So we should only need to set `GZ_` and use that.

Reply:
https://github.com/gazebosim/gz-cmake/pull/275/commits/5f4f308a2aa9b85caeaeab20d15cad690e29e545